sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -298,6 +298,17 @@ class SQLQuerySuite extends QueryTest with BeforeAndAfterAll with SQLTestUtils {
     )
   }
 
+  test("LOG") {
```

Comment:
let's remove this test case, and use selectExpr in DataFrameFunctionSuite to test the parser.